app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb

```diff
@@ -75,4 +83,5 @@ def delete_in_provider_queue
   COMMON_ATTRIBUTES = {}.freeze
   EXTRA_ATTRIBUTES = {}.freeze
   API_ATTRIBUTES = COMMON_ATTRIBUTES.merge(EXTRA_ATTRIBUTES).freeze
+  # TOWER_KIND = 'ssh'.freeze # default to `ssh` just like Tower does
```

Comment:
This type should never be created, right?  So I wouldn't include the constant here.